### PR TITLE
Updated Works ontology

### LINKED ISSUE #3: Document high-level domain model

### PR DESCRIPTION
**Pull request by ChristopherHilton**
Making changes in line with team meeting on ontologies, March 2017:
fixes #3

## What is this PR trying to achieve?
Synching in the changes that were made in March 2017 but not fully merged in.  
## Have the following been considered/are they needed?

- [ ] Tests?
- [ ] Docs?
- [x] Spoken to the right people?

### CODE REVIEW

**Comment by ChristopherHilton** [comment]
Having merged in March changes, added some new properties to bring in MIRO fields, and also corrected typo.

File: ontologies/works.ttl [hunk 1]
```diff
@@ -149,196 +181,167 @@ ww:startDate rdf:type owl:DatatypeProperty ;
...
+	rdfs:comment "A note of the conditions under which this work may be reproduced, including copyright issues, necessity to seek depositor permission, physical fragility, etc."@en ;
 	rdfs:domain ww:Work ;
 	rdfs:range rdf:langString ; 
-	rdf:isDefinedBy <TBC/works> .
-
+	rdf:isDefinedBy <http://www.wellcomecollection.org/ontologies/works> .
+	
+ww:wellcomeCollectionSection rdf:type owl:DatatypeProperty ;
+	rdfs:label "wellcomeCollectionSection"@en ;
+	rdfs:comment "this relates to works that are reproductions of other Wellcome Collection works, and records the section of the Wellcome Collection within which the original item is to be found. A note of the section of the Wellcome Collection within the original item is to be found."@en ;
```
Comment (alexwlchan) [inline ontologies/works.ttl]: nit: capitalise "this"

We also have repetition:

> and records the section of the Wellcome Collection within which the original item is to be found

> A note of the section of the Wellcome Collection within the original item is to be found

File: ontologies/works.ttl [hunk 2]
```diff
@@ -18,48 +18,52 @@ TBC
...
+	rdfs:comment "Format is used to provide a detailed expansion of a WorkType, in terms of physical characteristics, rendering requirements or both: thus, WorkType would state that a work comprises moving image material, whilst Format explains that it is a VHS video."@en ;
+	rdf:isDefinedBy <http://www.wellcomecollection.org/ontologies/works> .
 	
-ww:Location rdf:type owl:Class ;
-	rdfs:label "Location"@en ;
-	rdfs:comment "The place at which a work was published or otherwise created."@en ;
-	rdf:isDefinedBy <TBC/works> .
+ww:Place rdf:type owl:Class ;
+	rdfs:label "Place"@en ;
+	rdfs:comment "A physical location that relates in some way to a work: for example, the place at which a work was published or otherwise created."@en ;
```
Comment (ChristopherHilton) [inline ontologies/works.ttl]: Alas no: for the moment, like the link to Person authority, CALM just allows you to say that this place/person has "something to do" with the record.  This as you'll recall is something I've fed into the Requests for Enhancement and will be banging on about again at the Axiell conference in a couple of weeks!  For CALM, this sort of information will generally just be buried in the free-text description and this particular property won't be needed: it'll come into its own when we start looking at Sierra data, in which the MARC 260 field for publication details splits up the place, publisher and year in nice predictable ways.

File: ontologies/works.ttl (hunk 1 above)
Comment (ChristopherHilton) [inline ontologies/works.ttl]: Gosh, that was sloppy of me - will do.

File: ontologies/works.ttl (hunk 2 above)
Comment (pollecuttn) [inline ontologies/works.ttl]: With bib data something to keep in mind is the 008 might be better for date of publication (bytes 07-10; 11-14) and place of publication (bytes 15-17) http://www.lcweb.loc.gov/marc/bibliographic/bd008.html

File: ontologies/works.ttl (hunk 1 above)
Comment (ChristopherHilton) [inline ontologies/works.ttl]: That's now corrected in the repository.

File: ontologies/works.ttl [hunk 3]
```diff
@@ -70,77 +74,105 @@ ww:hasItem rdf:type owl:ObjectProperty ;
...
-	rdf:isDefinedBy <TBC/works> .
+	rdf:isDefinedBy <http://www.wellcomecollection.org/ontologies/works> .
 
 ww:hasPublisher rdf:type owl:ObjectProperty ;
 	rdfs:label "hasPublisher"@en ;
 	rdfs:comment "Relates a published work to its publisher."@en ;
 	rdfs:domain ww:Work ;
-	rdfs:range ww:Agent ;
-	rdf:isDefinedBy <TBC/works> .
+	rdfs:range wa:Agent ;
```
Comment (alexwlchan) [inline ontologies/works.ttl]: What’s wa?

Comment (ChristopherHilton) [inline ontologies/works.ttl]: That will be the Wellcome Agents ontology, when we come to talk about those.

**Pull request merged**